### DISCUSSION TITLE
Routing for different types of public transportation routes

### DISCUSSION THREAD

**Discussion by ahmetlii** [organicmaps/organicmaps#728] [Ideas]
Hello,
Support for routing tram, suburban rail, monorail, bus, and others would be good; especially when the subway is not the primary mode of transportation.
It would help to plan routes easier.
What do you think about it?

**Comment by biodranik** [comment]
We think it's definitely good. The only question is the complexity and resources to implement it.

**Reply by GLLM** [reply]
Hi

Since a few day, I've been wondering about the possibility for Organic Maps devs to leverage on the work of https://github.com/grote/Transportr, which has is a much more advanced routing/planning solution for public transportation...

Is this making sense in a way or another ? Or am I completely out of control ? :-)

Thanks
GLLM

**Reply by biodranik** [reply]
It makes sense. The main question is about the data, its license, and usage. Our main idea was to use as much OSM data as we can. Using other sources may add visual inconsistency to the map. But maybe there is a way to workaround it somehow. It would be great if someone will research it in more detail, as we don't have enough time to focus on public transport now.

**Comment by JVV1017** [comment]
I recently starting using OSM based map applications on my pixel 6, I don't have a car yet so I am still relying on buses and other means of transportation that aren't mainly subways, so I have to use OsmAnd~ for this but it is not good enough as well so I do have to keep using google maps but I was wondering if you guys have a plan to maybe somehow add more transportation options like buses especially? If you guys can't, its understandable as its tough to do so. I just happen to find the OsmAnd~ UI to be too complex and it doesn't look as user friendly as something like google, apple and also, with my small experience, organic maps.

**Reply by biodranik** [reply]
Yes, please check related issues. Any volunteers and help are welcome )
